### PR TITLE
fix: deploying compliment cmd

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds a compliment command to Ava

### CODE REVIEW

**Comment by karl-cardenas-coding** [comment]
@circa10a Wait...I forgot to update the README. 1 sec

File: commands/compliment.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    // Ensure message is intended for ava
+    if (!message.content.startsWith(avaPrefix)){
+      return;
+    }
+    const args = message.content.trim().split(/ +/g);
+    const userCmd = args[1];
+    const userToCompliment = args[2];
+
+    if (userCmd === command) {
+      if (!userToCompliment || !userToCompliment.startsWith('@')) {
```
Comment (circa10a) [inline commands/compliment.js:21]: lets just do `!userToCompliment`, some folks may not want to directly tag folks. like `ava compliment me`

**Comment by circa10a** [comment]
need to add `.vscode` and those other files to `.gitignore`

File: commands/compliment.js (hunk 1 above)
Comment (karl-cardenas-coding) [reply commands/compliment.js:21]: True, good point.

**Pull request merged**